docs/Flappi.html

```diff
@@ -83,7 +83,18 @@
   
 </div>
 
-<h2>Defined Under Namespace</h2>
+<h2>Overview</h2><div class="docstring">
+  <div class="discussion">
+    
+<p>rubocop:disable Metrics/LineLength</p>
```

Comment:
Why is this in here?

Reply:
That's a mystery. `yard` generated this file, and that rubocop:disable is nowhere in this codebase.

Reply:
i vote we delete this block

Reply:
or discard changes to this file. doesnt seem to be anything of signifiance

Reply:
Will delete the rubocop line 👍